src/Symfony/Component/Notifier/Bridge/Smsbox/SmsboxTransport.php

```diff
@@ -0,0 +1,161 @@
+    }
+
+    public function supports(MessageInterface $message): bool
+    {
+        return $message instanceof SmsMessage && (null === $message->getOptions() || $message->getOptions() instanceof SmsboxOptions);
+    }
+
+    protected function doSend(MessageInterface $message): SentMessage
+    {
+        if (!$message instanceof SmsMessage) {
+            throw new UnsupportedMessageTypeException(__CLASS__, SmsMessage::class, $message);
+        }
+
+        $phoneCleaned = preg_replace('/[^0-9+]+/', '', $message->getPhone());
+        if (!preg_match("/^(\+|)[0-9]{7,14}$/", $phoneCleaned)) {
+            throw new InvalidArgumentException('invalid number.');
+        }
+
+        $options = $message->getOptions()?->toArray() ?? [];
+        $options['dest'] = $phoneCleaned;
+        $options['msg'] = $message->getSubject();
+        $options['id'] = 1;
+        $options['usage'] = 'symfony';
+        $options['mode'] ??= $this->mode;
+        $options['strategy'] ??= $this->strategy;
+
+        if (SmsboxOptions::MESSAGE_MODE_EXPERT === $options['mode']) {
+            $options['origine'] = $options['sender'] ?? $this->sender;
+        }
+        unset($options['sender']);
+
+        if (isset($options['daysMinMax'])) {
+            $options['day_min'] = $options['daysMinMax'][0];
+            $options['day_max'] = $options['daysMinMax'][1];
+            unset($options['daysMinMax']);
+        }
+
+        if (isset($options['hoursMinMax'])) {
+            $options['hour_min'] = $options['hoursMinMax'][0];
+            $options['hour_max'] = $options['hoursMinMax'][1];
+            unset($options['hoursMinMax']);
+        }
+
+        if (isset($options['dateTime'])) {
+            if (isset($options['heure']) || isset($options['date'])) {
+                throw new InvalidArgumentException("You mustn't set the dateTime method along with date or hour methods.");
+            }
+
+            $options['date'] = $options['dateTime']->format('d/m/Y');
+            $options['heure'] = $options['dateTime']->format('H:i');
+            unset($options['dateTime']);
+        }
+
+        if (isset($options['variable'])) {
+            preg_match_all('%([0-9]+)%', $options['msg'], $matches);
+            $occurenceValMsg = $matches[0];
+            $occurenceValMsgMax = max($occurenceValMsg);
+
+            if ($occurenceValMsgMax != \count($options['variable'])) {
+                throw new InvalidArgumentException('You must have the same amount of index in your array as you have variable.');
+            }
+
+            $t = str_replace([',', ';'], ['%d44%', '%d59%'], $options['variable']);
+            $variableStr = implode(';', $t);
+            $options['dest'] .= ';'.$variableStr;
+            $options['personnalise'] = 1;
+            unset($options['variable']);
+        }
+
+        $endpoint = sprintf('https://%s/1.1/api.php', $this->getEndpoint());
+        $response = $this->client->request('POST', $endpoint, [
```

Comment:
```suggestion
        $response = $this->client->request('POST', sprintf('https://%s/1.1/api.php', $this->getEndpoint()), [
```

Lets remove the one time variable

Reply:
done 👍